content/culture.md

```diff
@@ -1,12 +1,64 @@
 # Culture
 
-## implementation methods that affect diversity and inclusion
+High level description of corporate culture what it means and its importance.
 
-Methodologies
+focus on the primarily used corporate and organizational culture
+- Heiarchical
+  - position/senority based
+  - chain of command
+  - distance between workers and managers
+  - merit based reward system
 
+How corporate culture affects social culture.
+
+What is valued?
+  - employees
+  - ideas/innovation
+  - work output
+How are issues resolved/handled?
+  - isolated and quiet
```

Comment:
"Isolated and quiet" and "free to broadcast concerns" could be conflict with each other

Reply:
Maybe we clarify that it's based on individual preference. Some people may want to have an offline conversation. Others are comfortable starting a wider discussion.

Reply:
These would be examples of conflict resolution. I think both have pros and cons that should be discussed in further depending on culture and individuals.